File: Curso_de_Programacion_Orientada_a_Objetos-POO/JS/Car.js
function Car(license, driver) {
	this.id;
	this.license = license;
	this.driver = driver;
	this.passenger;
}

Car.prototype.printDataCar = function () {
	console.log(
		"Car licence: " +
			this.license +
			" Driver: " +
			this.driver.name +
			" Documento: " +
			this.driver.document
	);
};

File: Curso_de_Programacion_Orientada_a_Objetos-POO/Java/Car.java
class Car {
  Integer id;
  String license;
  Account driver;
  Integer passenger;

  public Car(String license, Account driver) {
    this.license = license;
    this.driver = driver;

  }

  void printDataCar() {
    System.out.println(
        "Car licence: " + this.license + " Driver: " + this.driver.name + " Documento: " + this.driver.document
            + " Pasajeros: " + this.passenger);

  }
}
